src/anomalib/models/components/base/export_mixin.py

```diff
@@ -257,6 +252,95 @@ def to_openvino(
 
         return ov_model_path
 
+    def _compress_ov_model(
```

Comment:
This function seems to be a bit long. Would it be an idea to split it to improve the readability? Something like:

```python
def _compress_ov_model(...)
        ...
        if compression_type == CompressionType.INT8:
            model = nncf.compress_weights(model)
        elif compression_type == CompressionType.INT8_PTQ:
            model = _post_training_quantization(model, datamodule)
        elif compression_type == CompressionType.INT8_ACQ:
            model = _accuracy_control_quantization(model, datamodule, metric, task)
        else:
            raise ValueError("...")
        return model

```